types/beanstalkd-worker/index.d.ts

```diff
@@ -0,0 +1,248 @@
+// Type definitions for beanstalkd-worker 1.2
+// Project: https://github.com/burstable/node-beanstalkd-worker
+// Definitions by: Maxime LUCE <https://github.com/SomaticIT>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import BeanstalkdClient, { BeanstalkdJobState, BeanstalkdJobStats } from "beanstalkd";
+
+export default class BeanstalkdWorker {
```

Comment:
OK. So. I hate to say this, but [the package at runtime does not export a default to commonjs](https://runkit.com/embed/yij9h0iju8g3). The babel plugin (old version of babel?) they use to build hoists their `default` export into the entire module. So, I suggest:
```suggestion
export = BeanstalkdWorker;
class BeanstalkdWorker {
```
You'll need to lump all the other exported typed into a `namespace BeanstalkdWorker` to get them exported as well.

Reply:
Hello @weswigham,

I checked on the built version of the plugin and it's exported on `default` export.

```javascript
exports['default'] = BeanstalkdWorker;
module.exports = exports['default'];
```

So I think that the default export is the best way to type it.

Reply:
No, after a runtime check, you're right, it seems to be exported badly...

Reply:
I fixed it in commit https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39435/commits/07c7d5f73cfc901df332ca496b6cf1d745a3f082